test/integration/generated/datalakestorage/src/models/index.ts

```diff
@@ -180,7 +180,7 @@ export interface PathUpdateHeaders {
   /** User-defined properties associated with the file or directory, in the format of a comma-separated list of name and value pairs "n1=v1, n2=v2, ...", where each value is a base64 encoded string. Note that the string may only contain ASCII characters in the ISO-8859-1 character set. */
   properties?: string;
   /** When performing setAccessControlRecursive on a directory, the number of paths that are processed with each invocation is limited.  If the number of paths to be processed exceeds this limit, a continuation token is returned in this response header.  When a continuation token is returned in the response, it must be specified in a subsequent invocation of the setAccessControlRecursive operation to continue the setAccessControlRecursive operation on the directory. */
-  xMsContinuation?: string;
```

Comment:
This is because Style.camel("XMsContinuation", true, {}, 5} gives us XMsContinuation